src/CodeGen_Hexagon.cpp

```diff
@@ -46,17 +46,15 @@ CodeGen_Hexagon::CodeGen_Hexagon(Target t)
     } else {
         isa_version = 62;
     }
-    user_assert(!target.features_all_of(
-        {Halide::Target::HVX_128, Halide::Target::HVX_64}))
-        << "Cannot set both HVX_64 and HVX_128 at the same time.\n";
+    user_assert(target.has_feature(Target::HVX))
+        << "Creating a Codegen target for Hexagon without the hvx_128 or the hvx target feature.\n";
```

Comment:
Remove hvx_128 from this message?

Reply:
I left it in the message to indicate to users that ``hvx_128`` is still ok to use as a feature.